### ISSUE TITLE
No version of chokidar is available. Tried chokidar@2 and chokidar@3

### ISSUE THREAD

**Issue by zhouhanseng** [cypress-io/cypress-webpack-preprocessor#90]
- Operating System: macOS Catalina 10.15.1
- Cypress Version: 4.5.0
- Browser Version: chrome 81.0.4044.138

### Is this a Feature or Bug?
Bug

### Current behavior:
No version of chokidar is available. Tried chokidar@2 and chokidar@3

### Desired behavior:
Testing as usual

### How to reproduce:
```bash
$ git clone https://github.com/zhouhanseng/typescript-cypress-click-and-wait-element-appear-demo
$ npm install
$ npm run test:open
```

### Additional Info (images, stack traces, etc)
I think this is caused by a dependency issue, perhaps "chokidar-cli" version is too old. After I installed "chokidar-cli@2.1.0"，problem solved.

How about adding https://app.dependabot.com/ to this project?

**Comment by brian-mann** [comment]
`chokidar-cli` is a `devDependency` not a production dependency. This npm module does not use `chokidar` under the hood.

**Comment by zhouhanseng** [comment]
So, why problem solved after installed "chokidar-cli@2.1.0"？

**Comment by jennifer-shehane** [comment]
@zhouhanseng I'm not encountering any issues running the project as described. I do not have chokidar-cli installed. 

Please describe the error that you are seeing exactly when running `npm run test:open` within the provided repo. Screenshots are welcome. Until we have a reproducible example, this issue will be closed. We will reopen if one is provided.
